asv_bench/benchmarks/io/csv.py

```diff
@@ -146,10 +146,10 @@ def time_read_csv(self, bad_date_value):
 class ReadCSVSkipRows(BaseIO):
 
     fname = "__test__.csv"
-    params = [None, 10000]
-    param_names = ["skiprows"]
+    params = ([None, 10000], ["c"])
```

Comment:
Is this supposed to be c and python or just c?

Reply:
On current master we only have the default (I think that's `c`)

We have `c` and `python` in all the other benchmarks so it seems reasonable to add that here. If there's a reason we don't want that I'll revert